apps/elewa-website/src/app/app.module.ts

```diff
@@ -6,10 +6,11 @@ import { AppComponent } from './app.component';
 import { TranslocoRootModule } from './transloco-root.module';
 
 import { AppRoutingModule } from './app.routing';
+import { AppHeaderModule } from '@elewa-website/elements/layout/header';
 
```

Comment:
Fix the Order of imports. The routing comes last and domain imports come after angular imports.